src/test/java/com/google/sps/data/DatastoreScheduledInterviewTest.java

```diff
@@ -0,0 +1,185 @@
+      ScheduledInterview.create(
+          (long) -1,
+          new TimeRange(
+              Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+          "user@company.org",
+          "user2@mail.com");
+  private final ScheduledInterview scheduledInterview3 =
+      ScheduledInterview.create(
+          (long) -1,
+          new TimeRange(
+              Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+          "user3@company.org",
+          "user2@mail.com");
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig()
+              .setDefaultHighRepJobPolicyUnappliedJobPercentage(0));
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  // Test whether the scheduledInterview was added to datastore.
+  @Test
+  public void createsAndStoresEntity() {
+    tester.create(scheduledInterview1);
+    Entity entity = datastore.prepare(new Query("ScheduledInterview")).asSingleEntity();
+    ScheduledInterview storedScheduledInterview = tester.entityToScheduledInterview(entity);
+    ScheduledInterview copyScheduledInterview1 =
+        ScheduledInterview.create(
+            storedScheduledInterview.id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T17:00:10Z"), Instant.parse("2020-07-06T18:00:10Z")),
+            "user@company.org",
+            "user@mail.com");
+    Assert.assertEquals(true, storedScheduledInterview.equals(copyScheduledInterview1));
+  }
+
+  // Tests whether all scheduledInterviews for a particular user are returned.
+  @Test
+  public void getsAllScheduledInterviews() {
+    tester.create(scheduledInterview1);
+    tester.create(scheduledInterview2);
+    tester.create(scheduledInterview3);
+    List<ScheduledInterview> result = tester.getForPerson("user@company.org");
+    ScheduledInterview copyScheduledInterview1 =
+        ScheduledInterview.create(
+            result.get(0).id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T17:00:10Z"), Instant.parse("2020-07-06T18:00:10Z")),
+            "user@company.org",
+            "user@mail.com");
+    ScheduledInterview copyScheduledInterview2 =
+        ScheduledInterview.create(
+            result.get(1).id(),
+            new TimeRange(
+                Instant.parse("2020-07-06T19:00:10Z"), Instant.parse("2020-07-06T20:00:10Z")),
+            "user@company.org",
+            "user2@mail.com");
+
+    List<Boolean> actual = new ArrayList<Boolean>();
+    actual.add(copyScheduledInterview1.equals(result.get(0)));
+    actual.add(result.size() == 2);
+    actual.add(copyScheduledInterview2.equals(result.get(1)));
+
+    List<Boolean> expected = new ArrayList<Boolean>();
+    expected.add(true);
+    expected.add(true);
+    expected.add(true);
+
+    Assert.assertEquals(true, expected.equals(actual));
```

Comment:
So instead of doing this, you should make a list of your copyScheduledInterview objects and compare that (your expected) to the actual (result in this case).